qi-lib/flow.rkt

```diff
@@ -381,6 +383,11 @@ provide appropriate error messages at the level of the DSL.
               (apply append
                      (repeat n args))))]
   ;; [(_ ({~datum fanout} n:expr))
+  ;;  #'(lambda args
+  ;;      (apply values
+  ;;             (apply append
+  ;;                    (make-list n args))))]
+  ;; [(_ ({~datum fanout} n:expr))
```

Comment:
Did this end up being slower (based on the commenting)?

Reply:
Yeah it did, oddly enough. I'm writing a longer comment with more details.